models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -169,6 +171,16 @@ with session_firsts as (
         {%- endfor %}
     {%- endif %}
     from {{ ref('snowplow_web_base_events_this_run') }}
+    left join
+        (
+            select
+                domain_sessionid,
+                event_name,
+                count(*) as event_count
+            from
+                {{ ref('snowplow_web_base_events_this_run') }}
+            group by domain_sessionid, event_name
+        ) using (domain_sessionid)
```

Comment:
We're going to have to do this a different way, it currently errors due to duplicate keys (which I think we could fix with some renaming), but the main issue is this is a 1:N join that throws off all the other aggregations. I've got an in-line solution I'll replace it with.